test/functional/services/dashboard/visualizations.ts

```diff
@@ -80,6 +82,7 @@ export class DashboardVisualizationsService extends FtrService {
       await this.dashboard.switchToEditMode();
     }
     await this.dashboardAddPanel.addSavedSearch(name);
+    await this.dashboard.waitForRenderComplete();
```

Comment:
Would it make more sense to add `waitForRenderComplete` in `dashboardAddPanel.addSavedSearch` call?

Reply:
looking at `addSavedSearch` closer, I think you are right to not update that method and keep things as they are